staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -94,3 +100,13 @@ func (a *AdmissionOptions) Validate() []error {
 	errs := []error{}
 	return errs
 }
+
+// SetDefaultPlugins sets the PluginNames field if it was not provided.
+// The field is populated from the hard coded list of default plugin names.
+// Note: This method MUST be called after AddFlags
+func (a *AdmissionOptions) SetDefaultPlugins() {
```

Comment:
At least for now. Later on they could be provided from the command line or a dynamic config.